lib/utils.dart

```diff
@@ -16,42 +16,33 @@ String getString(
   String name,
   String message, {
   required bool isInteractive,
+  required logger.Logger logger,
   String? desc,
   String? Function(String)? validate,
 }) {
   var value = results[name] as String?;
   if (!isInteractive) {
     if (value == null || value.isEmpty) {
-      print('Missing parameter $name is required.');
-      exit(1);
+      logger.info('Missing parameter $name is required.');
```

Comment:
this info message and others probably should use `logger.err` and write to stderr, but the tests from `./scripts/test.sh` fail

Reply:
Feel free to do breaking changes, maybe the test could be updated to accommodate for the change?

Reply:
yeah def feel free to change the test to expect stderr in this case!